engines/sword25/gfx/staticbitmap.cpp

```diff
@@ -124,10 +124,6 @@ uint StaticBitmap::getPixel(int x, int y) const {
 	assert(pResource->getType() == Resource::TYPE_BITMAP);
 	BitmapResource *pBitmapResource = static_cast<BitmapResource *>(pResource);
 	uint result = pBitmapResource->getPixel(x, y);
-	// Convert to LUA-ready format
-	byte a;
-	a = result & 0xff;
-	result = (result >> 8) | (a << 24);
```

Comment:
Are you sure about this change? This reverts dbb473752b

Reply:
The conversion is now done in `SWImage::getPixel()` instead, so it should still work. As mentioned above, though, I'm not sure how to test this properly.